code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -611,7 +611,15 @@ GLOBAL_LIST_INIT(brass_recipes, list (
 	new /datum/stack_recipe/window("directional brass window", /obj/structure/window/reinforced/clockwork, time = 0 SECONDS, on_floor = TRUE, window_checks = TRUE),
 	new /datum/stack_recipe/window("fulltile brass window", /obj/structure/window/reinforced/clockwork/fulltile, 2, time = 0 SECONDS, on_floor = TRUE, window_checks = TRUE),
 	new /datum/stack_recipe("brass chair", /obj/structure/chair/brass, 1, time = 0 SECONDS, one_per_turf = TRUE, on_floor = TRUE),
+	new /datum/stack_recipe_list("pews", list(
+		new /datum/stack_recipe("brass pew (middle)", /obj/structure/chair/sofa/pew/clockwork, 5, one_per_turf = TRUE, on_floor = TRUE),
```

Comment:
Wooden pews are mare out of a single sheet, Five seems to be a bit much

Reply:
Wooden pews cost 5 sheets of wood to make, that is what I based the cost on.

Reply:
Seems the wiki is incorrect! It said they were one there. I'll dismiss my comments and update the wiki!